src/v2/view-builder/utils/ChallengeViewUtil.js

```diff
@@ -83,5 +83,24 @@ export function doChallenge(view) {
         Util.redirect(deviceChallenge.href);
       }
     }));
+    break;
+  case Enums.APP_LINK_CHALLENGE:
+    view.title = loc('appLink.title', 'login');
+    view.add(View.extend({
+      className: 'app-link-content',
```

Comment:
Curious why we need a `className` here? Seems like we can simplify this to:

```suggestion
    view.add(loc('appLink.content', 'login'));
```

Reply:
i'm following https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/utils/ChallengeViewUtil.js#L74 as they are very similar. While i didn't find specific use case for the classname, it's being used in tests https://github.com/okta/okta-signin-widget/blob/master/test/unit/spec/v2/view-builder/utils/ChallengeViewUtil_spec.js#L118 probably @jhe-okta has a better idea